Week-2/part1/style.css

```diff
@@ -0,0 +1,103 @@
+/* CSS Reset */
+/*
+  1. Use a more-intuitive box-sizing model.
+*/
+*, *::before, *::after {
+    box-sizing: border-box;
+  }
+  /*
+    2. Remove default margin
+  */
+  * {
+    margin: 0;
+  }
+  /*
+    3. Allow percentage-based heights in the application
+  */
+  html, body {
```

Comment:
Here you can also make your font-size consistent and then use units like rem and em.